CHANGES.md

```diff
@@ -1,9 +1,20 @@
-## In the next release...
+## 0.8.6 2017-01-19
 
 * Add experimental StatsD telemeter
-* Add a recent requests log in the linkerd admin dashboard
-* Admin dashboard now works if served at a non-root url
+* Admin dashboard
+  * Add a log of recent requests
+  * Admin dashboard now works if served at a non-root url
```

Comment:
Tiny nit: no need to repeat "Admin dashboard" in this sub-bullet. Maybe just "Now works if served at a non-root url".